beignner_Level(krishita30j).js

```diff
@@ -0,0 +1,53 @@
+//1. Do the inverse of method "push()".
+
+var arr = [12, 45, 67, 23, 78];
+arr.unshift(100);
+console.log(arr);
+//Output: [100, 12, 45, 67, 23, 78]
+
+
+//2. Inverse of the output.
+
+const x = [];
+const y = 1;
+
+console.log(x == y ? false : true);
+//Output: true
+
+
+//3. Variable's scope
+
+const a = 10;
+
+function foo() {
+    var a = 20;
+}
+console.log(a);
+
+foo();
+//Output: 10
+
+
+//4. Array's value
+
+var fruits = ['Lemon', 'Pomelo', 'Kumquat', 'Yuzu', 'Blood orange'];
+
+fruits.forEach((currentValue) => {
```

Comment:
There is **JS** ***for loop*** called ****for-of****.
It iterates an array and grabs the value itself.
```
for(i of fruits) {
    console.log(i);
}
```